osu.Game/Localisation/AimErrorMeterStrings.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class AimErrorMeterStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.HUD.PositionMeterStrings";
+
+        /// <summary>
+        /// "Judgment position size."
+        /// </summary>
+        public static LocalisableString JudgmentSize => new TranslatableString(getKey(@"judgement_line_thickness"), "Judgment position size.");
```

Comment:
You should avoid dots here to be consistent with the rest UI same elements.
```suggestion
        public static LocalisableString JudgmentSize => new TranslatableString(getKey(@"judgement_line_thickness"), "Judgment position size");
```